src/libraries/System.Runtime.Serialization.BinaryFormat/src/System/Runtime/Serialization/BinaryFormat/BinaryArrayType.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Runtime.Serialization.BinaryFormat;
+
+/// <summary>
+/// Binary array type.
```

Comment:
https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Enum

```suggestion
/// Indicates the kind of an array for an NRBF BinaryArray record.
```